Flake8 fails to lint phone number regex

Builds are failing the flake8 check because of the following:

```
./timetracker/vms/forms.py:156:7: W605 invalid escape sequence '\s'
./timetracker/vms/forms.py:156:12: W605 invalid escape sequence '\d'
./timetracker/vms/forms.py:156:13: W605 invalid escape sequence '\+'
./timetracker/vms/forms.py:156:17: W605 invalid escape sequence '\d'
./timetracker/vms/forms.py:156:25: W605 invalid escape sequence '\d'
./timetracker/vms/forms.py:156:35: W605 invalid escape sequence '\d'
./timetracker/vms/forms.py:156:38: W605 invalid escape sequence '\d'
./timetracker/vms/forms.py:156:44: W605 invalid escape sequence '\s'
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -10,7 +10,7 @@ env:
 
 install:
   - pip install --upgrade pipenv
-  - pipenv install --dev
+  - pipenv sync --dev
 
 script:
   - pipenv run flake8

--- a/timetracker/vms/forms.py
+++ b/timetracker/vms/forms.py
@@ -114,8 +114,8 @@ class CreateStaffAgencyForm(forms.Form):
     email_field = forms.EmailField(required=True)
     name_field = forms.CharField(required=True)
     notes_field = forms.CharField(required=False, widget=forms.Textarea)
-    regex = '^\s*(?:\+?(\d{1,3}))?[-. (]*(\d{3})[-'\
-        '. )]*(\d{3})[-. ]*(\d{4})(?: *x(\d+))?\s*$'
+    regex = r'^\s*(?:\+?(\d{1,3}))?[-. (]*(\d{3})[-'\
+        r'. )]*(\d{3})[-. ]*(\d{4})(?: *x(\d+))?\s*$'
     phone_number_field = forms.RegexField(regex=regex, required=False)
 
     def save(self, user):

